analyzers/src/SonarAnalyzer.CFG/ShimLayer/AnalysisContext/SymbolStartAnalysisContextWrapper.cs

```diff
@@ -76,6 +76,10 @@ static SymbolStartAnalysisContextWrapper()
         // receiverParameter => ((symbolStartAnalysisContextType)receiverParameter)."propertyName"
```

Comment:
Is this comment forgotten?

Reply:
No it's fine: this is the C#-rendered code that is built via LINQ Expression Tree.